project/src/main/java/com/google/sps/servlets/BlobUrlServlet.java

```diff
@@ -14,7 +14,12 @@
 public class BlobUrlServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String uploadUrl = BlobstoreServiceFactory.getBlobstoreService().createUploadUrl("/blob-handler");
+    String pictureType = request.getParameter("picture-type");
+    String servletUrl = String.format("/%s-blob-handler", pictureType);
+    if (pictureType.equals("event")) {
```

Comment:
May want to throw an error here if pictureType is null.

Reply:
Done, thanks for catching